package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "25.7_uwierzytelnianie_apk_za_pomoca_google",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u00002\u00005\u0000.\u00007\u0000-\u0000u\u0000w\u0000i\u0000e\u0000r\u0000z\u0000y\u0000t\u0000e\u0000l\u0000n\u0000i\u0000a\u0000n\u0000i\u0000e\u0000-\u0000a\u0000p\u0000k\u0000-\u0000z\u0000a\u0000-\u0000p\u0000o\u0000m\u0000o\u0000c\u0000a\u0000-\u0000g\u0000o\u0000o\u0000l\u0000e\u0000\r\u0000 \u0000",
```

Comment:
A co to za ładny opis, proponuje usunąć go całkiem lub wprowadzić bardziej czytelny.

Reply:
taki opis tworzy mi się automatycznie gdy tworzę 'npm init' i zostawiam puste pole przy description